src/implementations/twig/components/tabs/tabs.html.twig

```diff
@@ -92,6 +104,7 @@
             transform: 'rotate-180'
           },
           extra_classes: 'ecl-tabs__toggle',
+          extra_attributes: [{ name: 'tabindex', value: '-1' }],
```

Comment:
i understand that we are introducing navigation through keyboard arrows, but why disabling the standard navigation via Tab..? It feels really weird to access the first tab with a Tab and then being moved out of the tabs typing again Tab. This way the user really have to know that the only way of navigating our tabs would be using arrows.
Also because the standard navigation would be prefectly working in most of the viewports and only in mobile our "alternative" navigation would really be helpful..

Reply:
This is basically the request, to only be able to navigate with arrows and thus be able to go to the content more quickly without having to tab all tabs. https://www.w3.org/WAI/ARIA/apg/example-index/tabs/tabs-manual.html

Reply:
i don't understand, really..

Reply:
We can discuss this on the next weekly if you want ;)